src/textfont.ts

```diff
@@ -9,33 +9,93 @@
 import { Vex } from './vex';
 import { PetalumaScriptTextMetrics } from './fonts/petalumascript_textmetrics';
 import { RobotoSlabTextMetrics } from './fonts/robotoslab_textmetrics';
+export interface TextFontMetrics {
+  advanceWidth: number;
+  ha: number;
+}
+
+export interface TextFontGlyph {
```

Comment:
This looks just like the other glyph metrics. Can we share?

Reply:
Yes, with minor changes in `FontGlyph` done.